bepasty_cli/cli.py

```diff
@@ -30,6 +31,18 @@
 LIFETIME_MAPPING = dict(zip(LIFETIME_CHOICES, LIFETIME_NAMES))
 
 
+class LifetimeParamType(click.ParamType):
+    name = 'lifetime'
+    lifetime_regex = re.compile(r'|'.join(r'^([1-9]\d*?) ?({})$'.format(choice) for choice in LIFETIME_CHOICES))
```

Comment:
hmm, we just need to disallow that someone gives e.g. `0d`, as giving a zero time (no matter what unit) is invalid.

but this maybe does not need to get addressed by the regex, can be checked later.